scalableminds/webknossos
# Fewer error messages in view mode

This PR reduces the possible error messages displayed in view only mode.
The two errors that are now ignored in view only mode are:
- The error that is show when the updating the tracing fails.
- The error that is show when saving the users dataset settings fails.

### URL of deployed dev instance (used for testing):
- https://lesserrormessagesinviewmode.webknossos.xyz

### Steps to test:
- I only found out how to reproduce the second error (locally):
  - Open a dataset in view mode and when it is loaded kill your local server.
  - The open the dataset settings an eg. enable/disable the "Render Isosurfaces (Beta)"-setting. This should not trigger an error.
  - If you try the same on the master, the tracing should enter the "error state" and an alert should be displayed. 

### Issues:
- fixes #5046

------
